css/master.css

```diff
@@ -334,3 +334,93 @@
   padding-top: 30%;
   text-align: center;
 }
+
+.contact {
+  margin-top: var(--section-main-padding);
+  margin-bottom: var(--section-main-padding);
+}
+
+.contact .contact-header {
+  font-size: 32px;
+  font-weight: bold;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 60px;
+}
+
+.contact form .text-email input {
+  border: none;
+  background-color: white;
+  width: 100%;
+  border-bottom: solid 1px grey;
+  padding: 5px;
+  margin-bottom: 15px;
+  text-indent: 5px;
+  font-family: "Inter", sans-serif;
+}
+
+.contact form textarea {
+  width: 100%;
+  height: 114px;
+  font-family: inherit;
+  font-size: 100%;
+  line-height: 1.15;
+  margin: 0;
+  border: none;
+  background-color: #fbf8f7;
+}
+
+.contact form textarea::placeholder {
+  color: black;
+  font-weight: 500;
+  font-size: 15px;
+  text-indent: 10px;
+  line-height: 30px;
+  font-family: "Inter", sans-serif;
+}
+
+.contact form .submit {
+  display: flex;
+  justify-content: center;
+}
+
+.contact form .submit input {
+  padding: 13px;
```

Comment:
- [ ] The Button interactions such as hover, active (when it's pressed), are styled as shown in the Figma template. Please make this button style effect  ⬇️ 

![buttonfine](https://user-images.githubusercontent.com/79058364/165497321-978a2540-9ff0-4852-8c2a-adad87378135.png)